package/Dockerfile

```diff
@@ -55,7 +55,9 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 102.1.0+up0.5.0
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=102.2.0+up0.8.0-rc.6
+# Deprecated in favor of CATTLE_FLEET_VERSION.
```

Comment:
shouldn't the comment be above the line with the deprecated variable, as it is for `CATTLE_RANCHER_WEBHOOK_MIN_VERSION`?